src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,34 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterAactions } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const filter = useAppSelector(state => state.filter);
+
+  const onStatusSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
```suggestion
  const handleStatusSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
```

Reply:
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364